controllers/host/platformnetworks.go

```diff
@@ -595,6 +595,15 @@ func (r *HostReconciler) ReconcilePlatformNetworkExpected(client *gophercloud.Se
 		case cloudManager.OAMNetworkType,
 			cloudManager.MgmtNetworkType,
 			cloudManager.AdminNetworkType:
+
+			// Reconciliation is not expected for management network on
+			// AIO-DX / Standard setups.
+			if system_info.SystemMode != cloudManager.SystemModeSimplex &&
```

Comment:
Why do we want to check the system mode? Isn't it expected to be rejected by the sysinv API?